spec/classes/patroni_spec.rb

```diff
@@ -402,5 +402,27 @@
         end
       end
     end
+    context 'custom_pip_provider => undef' do
```

Comment:
The `virtualenv` attribute would also be affected

Reply:
Is this a comment on the context name? Or something in the context block?